cohortextractor/patients.py

```diff
@@ -1378,39 +1382,91 @@ def with_test_result_in_sgss(
-            "SARS-CoV-2".
+        pathogen: pathogen we are interested in. Only SARS-CoV-2 results are
+            included in our data extract so this will throw an error if the
+            specified pathogen is anything other than "SARS-CoV-2".
         test_result: must be one of "positive", "negative" or "any"
-        on_or_before: date of interest as a string with the format `YYYY-MM-DD`. Filters results to
-            on or before the given date. The default value is `None`.
-        on_or_after: date of interest as a string with the format `YYYY-MM-DD`. Filters results to
-            on or after the given date. The default value is `None`.
-        between: two dates of interest as a list with each date as a string with the format `YYYY-MM-DD`.
-            Filters results to between the two dates provided. The default value is `None`.
-        find_first_match_in_period: a boolean that indicates if the data returned is first event
-            if there are multiple matches within the time period
-        find_last_match_in_period: a boolean that indicates if the data returned is last event
-            if there are multiple matches within the time period
-        returning: a string indicating what type of value should be returned. The options are limited to binary_flag
-            (which indicates if they have had the an event or not) and date (which indicate date of event and used
-            with either find_first_match_in_period or find_last_match_in_period)
-        date_format: a string detailing the format of the dates to be returned. It can be `YYYY-MM-DD`,
-            `YYYY-MM` or `YYYY` and wherever possible the least disclosive data should be returned. i.e returning
-            only year is less disclosive than a date with day, month and year.
-        return_expectations: a dictionary defining the incidence and distribution of expected value
-            within the population in question.
+        on_or_before: date of interest as a string with the format
+            `YYYY-MM-DD`. Filters results to on or before the given date. The
+            default value is `None`.
+        on_or_after: date of interest as a string with the format `YYYY-MM-DD`.
+            Filters results to on or after the given date. The default value is
+            `None`.
+        between: two dates of interest as a list with each date as a string
+            with the format `YYYY-MM-DD`.  Filters results to between the two
+            dates provided. The default value is `None`.
+        find_first_match_in_period: a boolean that indicates if the data
+            returned is first event if there are multiple matches within the
+            time period
+        find_last_match_in_period: a boolean that indicates if the data
+            returned is last event if there are multiple matches within the
+            time period
+        restrict_to_earliest_specimen_date: a boolean indicating whether to use
```

Comment:
I find this a bit awkward, specially when describing it in the "exposition" bit above.

Would `treat_every_specimen_as_new_episode` be any better. Just spitballin